apps/etterlatte-beregning/src/main/kotlin/avkorting/InntektAvkortingService.kt

```diff
@@ -60,74 +65,72 @@ object InntektAvkortingService {
         beregninger: List<Beregningsperiode>,
         avkortingsperioder: List<Avkortingsperiode>
     ): List<AvkortetYtelse> {
-        val regelgrunnlag = periodisertBruttoYtelseOgAvkorting(beregninger, avkortingsperioder)
-        return regelgrunnlag.map { grunnlag ->
-            val resultat = avkorteYtelse.eksekver(KonstantGrunnlag(grunnlag), grunnlag.periode)
-            when (resultat) {
-                is RegelkjoeringResultat.Suksess -> {
-                    val tidspunkt = Tidspunkt.now()
-                    resultat.periodiserteResultater.map { periodisertResultat ->
-                        AvkortetYtelse(
-                            periode = Periode(
-                                fom = YearMonth.from(periodisertResultat.periode.fraDato),
-                                tom = periodisertResultat.periode.tilDato?.let { YearMonth.from(it) }
-                            ),
-                            ytelseEtterAvkorting = periodisertResultat.resultat.verdi,
-                            avkortingsbeloep = grunnlag.avkorting.verdi,
-                            ytelseFoerAvkorting = grunnlag.bruttoYtelse.verdi,
-                            tidspunkt = tidspunkt,
-                            regelResultat = periodisertResultat.toJsonNode(),
-                            kilde = Grunnlagsopplysning.RegelKilde(
-                                navn = kroneavrundetInntektAvkorting.regelReferanse.id,
-                                ts = tidspunkt,
-                                versjon = periodisertResultat.reglerVersjon
-                            )
+        val regelgrunnlag = PeriodisertAvkortetYtelseGrunnlag(
+            beregninger = periodiserteBeregninger(beregninger),
+            avkortinger = periodiserteAvkortinger(avkortingsperioder)
+        )
+        val resultat = avkorteYtelse.eksekver(regelgrunnlag, RegelPeriode(beregninger[0].datoFOM.atDay(1)))
```

Comment:
kan det være en idè å kalle det for `beregningsperioder` i stedet? Så blir det litt tilsvarende ordlyd som `avkortingsperioder`.